data/json/items/gun/223.json

```diff
@@ -27,64 +27,28 @@
     "melee_damage": { "bash": 12 }
   },
   {
-    "id": "ar_pistol",
-    "copy-from": "rifle_semi",
+    "id": "modular_ar_pistol",
+    "copy-from": "pistol_base",
+    "//": "Roughly based on https://www.cheaperthandirt.com/del-ton-lima-ar-15-semi-auto-pistol-5.56-nato-7.5-barrel-30-rounds-m-lok-handguard-black/FC-848456002687.html, with the upper subtracted.",
     "looks_like": "modular_ar15",
     "type": "GUN",
     "name": { "str": "AR pistol" },
-    "description": "A compact, 7.5 inch barrel version of the classic AR-15 design, commercially marketed as a home defense weapon.",
-    "weight": "2267 g",
-    "volume": "1758 ml",
-    "longest_side": "584 mm",
-    "barrel_length": "191 mm",
-    "price": "914 USD",
+    "description": "The common AR-15 platform, with its stock removed and buffer tube covered by a sleeve.  AR pistols lack stocks, or have arm braces due to restrictions on short barreled rifles, and so without a stock, they're legally classified as a pistol.",
+    "weight": "1043 g",
+    "volume": "748 ml",
+    "longest_side": "373 mm",
+    "price": "245 USD",
     "price_postapoc": "35 USD",
+    "to_hit": { "grip": "none", "length": "hand", "surface": "any", "balance": "clumsy" },
```

Comment:
Considering the lower receiver still has _a grip_ on it, shouldn't this be solid or weapon 🤔